go/libkb/globals.go

```diff
@@ -515,6 +516,12 @@ func (g *GlobalContext) GetUserEKBoxStorage() UserEKBoxStorage {
 	return g.userEKBoxStorage
 }
 
+func (g *GlobalContext) GetTeamEKBoxStorage() TeamEKBoxStorage {
+	g.cacheMu.RLock()
+	defer g.cacheMu.RUnlock()
+	return g.teamEKBoxStorage
```

Comment:
I'm not sure the original intention, i just modeled the getter/setter for this as the other caches are.. should we ditch it?

Reply:
Keeping consistent with existing code seems fine. I just want to make sure we've both thought about it, so that we can keep an eye out for code that assumes these locks actually do something :-D

Reply:
And of course, our support for safe pointer operations on 16-bit microcontrollers will be...nonzero :+1: